classes/ValidFormBuilder/Area.php

```diff
@@ -552,7 +577,7 @@ public function getValue($intCount = null)
      */
     public function getId()
     {
-        return $this->getName() . "_wrapper";
+        return $this->getName() . "_vf_wrapper";
```

Comment:
Potential backwards compatibility break! Is it really necessary?
It would mean that we have to raise the major version...